db/seeds.rb

```diff
@@ -5,9 +5,27 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+Category.create(name: "drama")
```

Comment:
You could add bang to create, to see the error message in case the seeding would fail. Saying from my own experience, it saves time trying to figure out what was wrong :)